update_versions.conf

```diff
@@ -30,17 +30,24 @@ targetfile: src/setup.py
 # The following file does not exist in the repo as a static file
 # It is generated at build time
 sourcefile: .chart_version
-tag: @CHART_VERSION@
+tag: 0.0.0-chart
```

Comment:
Crud, so now we have to remember to increment all of these versions when we bump a version?

Reply:
No. The build pipeline and buildprep.sh inject the correct values into these files (or the copies of the files). We still have to increment and release with a tag.